product_pricelist_tier/models/product_pricelist_item.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+from odoo.addons import decimal_precision as dp
+from odoo.exceptions import ValidationError
+
+
+class PricelistItem(models.Model):
+
+    _inherit = 'product.pricelist.item'
+
+    compute_price = fields.Selection(
+        selection_add=[
+            ('tiered', 'Tiered Price'),
+        ],
+    )
+    tiered_price = fields.Float(
+        string='Tiered Price',
+        digits=dp.get_precision('Product Price'),
+        help='The desired price at minimum quantity you specify. '
+        'e.g. You want 10 bowls to cost a total of $20. '
+        'Set this field to 20, and set minimum quantity to 10. The '
+        'price discount will automatically be calculated for you.',
+        store=True,
+    )
+
+    @api.onchange('compute_price')
```

Comment:
This should also be onchange `tiered_price` - you are using it in the calculations